recipes-debug/vendor-tools/files/network_setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
Please can this file have an RDK Apache header, and please can network-setup_1.0.bb use LICENSE="Apache-2.0" instead, with LIC_FILES_CHKSUM referring to an Apache license instead.